packages/cli/src/bin.ts

```diff
@@ -84,12 +84,12 @@ async function main(): Promise<number> {
     {
       init: Init.new(),
       platform: Platform.$.new({
-        login: Platform.Login.new(),
-        logout: Platform.Logout.new(),
         workspace: Platform.Workspace.$.new({
           show: Platform.Workspace.Show.new(),
         }),
         auth: Platform.Auth.$.new({
+          login: Platform.Login.new(),
+          logout: Platform.Logout.new(),
```

Comment:
should we also change the methods to be:
```suggestion
          login: Platform.Auth.Login.new(),
          logout: Platform.Auth.Logout.new(),
```
so it can be consistent with the other commands and hierarchies

Reply:
Yes absolutely! I moved the files and also updated some old copy references